Description: Fix FTBFS on hppa by disabling threads.
Author: Felix Geyer <debfx-pkg@fobos.de>

--- a/src/kmk/Makefile.kmk
+++ b/src/kmk/Makefile.kmk
@@ -252,6 +252,9 @@ kmk_SOURCES = \
 kmk_SOURCES.win = \
 	w32/subproc/sub_proc.c
 
+kmk_DEFS.hppa32 = CONFIG_WITHOUT_THREADS
+kmk_DEFS.hppa64 = CONFIG_WITHOUT_THREADS
+
 kmk_DEFS.gnukfreebsd = CONFIG_WITHOUT_THREADS
 kmk_DEFS.freebsd.x86 = CONFIG_WITHOUT_THREADS
 
