Add patch to fix issues on ppc64el
Index: kbuild/kBuild/env.sh
===================================================================
--- kbuild.orig/kBuild/env.sh	2014-12-05 15:57:43.546393368 +0530
+++ kbuild/kBuild/env.sh	2014-12-05 15:57:43.546393368 +0530
@@ -351,6 +351,9 @@
         ppc64|powerpc64)
             KBUILD_HOST_ARCH='ppc64'
             ;;
+        ppc64le)
+            KBUILD_HOST_ARCH='ppc64el'
+            ;;
         mips32|mips)
             KBUILD_HOST_ARCH='mips32'
             ;;
Index: kbuild/kBuild/header.kmk
===================================================================
--- kbuild.orig/kBuild/header.kmk	2014-12-05 15:57:43.546393368 +0530
+++ kbuild/kBuild/header.kmk	2014-12-05 15:57:43.546393368 +0530
@@ -189,8 +189,8 @@
 # build types. (PORTME)
 #
 KBUILD_OSES      := darwin dos dragonfly freebsd gnukfreebsd haiku l4 linux netbsd nt openbsd os2 solaris win os-agnostic
-KBUILD_ARCHES    := x86 amd64 sparc32 sparc64 s390 s390x ppc32 ppc64 mips32 mips64 ia64 hppa32 hppa64 arm alpha sh
-KBUILD_ARCHES_64 := amd64 sparc64 s390x ppc64 mips64 ia64 hppa64 alpha
+KBUILD_ARCHES    := x86 amd64 sparc32 sparc64 s390 s390x ppc32 ppc64 ppc64el mips32 mips64 ia64 hppa32 hppa64 arm alpha sh
+KBUILD_ARCHES_64 := amd64 sparc64 s390x ppc64 ppc64el mips64 ia64 hppa64 alpha
 KBUILD_ARCHES_32 := x86 sparc32 s390 ppc32 mips32 hppa32 arm sh
 
 
