Index: kbuild-0.1.9998svn2695+dfsg/kBuild/env.sh
===================================================================
--- kbuild-0.1.9998svn2695+dfsg.orig/kBuild/env.sh
+++ kbuild-0.1.9998svn2695+dfsg/kBuild/env.sh
@@ -351,6 +351,9 @@ if test -z "$KBUILD_HOST_ARCH"; then
         ppc64|powerpc64)
             KBUILD_HOST_ARCH='ppc64'
             ;;
+        ppc64le)
+            KBUILD_HOST_ARCH='ppc64el'
+            ;;
         mips32|mips)
             KBUILD_HOST_ARCH='mips32'
             ;;
Index: kbuild-0.1.9998svn2695+dfsg/kBuild/header.kmk
===================================================================
--- kbuild-0.1.9998svn2695+dfsg.orig/kBuild/header.kmk
+++ kbuild-0.1.9998svn2695+dfsg/kBuild/header.kmk
@@ -189,8 +189,8 @@ KBUILD_BLD_TYPES := release profile debu
 # build types. (PORTME)
 #
 KBUILD_OSES      := darwin dos dragonfly freebsd gnukfreebsd haiku l4 linux netbsd nt openbsd os2 solaris win os-agnostic
-KBUILD_ARCHES    := x86 amd64 sparc32 sparc64 s390 s390x ppc32 ppc64 mips32 mips64 ia64 hppa32 hppa64 arm alpha sh
-KBUILD_ARCHES_64 := amd64 sparc64 s390x ppc64 mips64 ia64 hppa64 alpha
+KBUILD_ARCHES    := x86 amd64 sparc32 sparc64 s390 s390x ppc32 ppc64 ppc64el mips32 mips64 ia64 hppa32 hppa64 arm alpha sh
+KBUILD_ARCHES_64 := amd64 sparc64 s390x ppc64 ppc64el mips64 ia64 hppa64 alpha
 KBUILD_ARCHES_32 := x86 sparc32 s390 ppc32 mips32 hppa32 arm sh
 
 
