--- a/kBuild/env.sh
+++ b/kBuild/env.sh
@@ -372,6 +372,9 @@
         arm|armv4l|armv5tel|armv5tejl|armv7l)
             KBUILD_HOST_ARCH='arm'
             ;;
+        aarch64)
+            KBUILD_HOST_ARCH='arm64'
+            ;;
         alpha)
             KBUILD_HOST_ARCH='alpha'
             ;;
--- a/kBuild/header.kmk
+++ b/kBuild/header.kmk
@@ -189,8 +189,8 @@
 # build types. (PORTME)
 #
 KBUILD_OSES      := darwin dos dragonfly freebsd gnukfreebsd haiku l4 linux netbsd nt openbsd os2 solaris win os-agnostic
-KBUILD_ARCHES    := x86 amd64 sparc32 sparc64 s390 s390x ppc32 ppc64 ppc64el mips32 mips64 ia64 hppa32 hppa64 arm alpha sh
-KBUILD_ARCHES_64 := amd64 sparc64 s390x ppc64 ppc64el mips64 ia64 hppa64 alpha
+KBUILD_ARCHES    := x86 amd64 sparc32 sparc64 s390 s390x ppc32 ppc64 ppc64el mips32 mips64 ia64 hppa32 hppa64 arm arm64 alpha sh
+KBUILD_ARCHES_64 := amd64 sparc64 s390x ppc64 ppc64el mips64 ia64 hppa64 arm64 alpha
 KBUILD_ARCHES_32 := x86 sparc32 s390 ppc32 mips32 hppa32 arm sh
 
 
--- a/src/lib/k/kDefs.h
+++ b/src/lib/k/kDefs.h
@@ -192,6 +192,8 @@
 #  define K_ARCH    K_ARCH_ALPHA
 # elif defined(__arm__) || defined(__arm32__)
 #  define K_ARCH    K_ARCH_ARM_32
+# elif defined(__aarch64__)
+#  define K_ARCH    K_ARCH_ARM_64
 # elif defined(__hppa__) && defined(__LP64__)
 #  define K_ARCH    K_ARCH_PARISC_64
 # elif defined(__hppa__)
