Description: Support also armv8 with 32 bit environment
Author: Gianfranco Costamagna <locutusofborg@debian.org>
Forwarded: on irc to <bird>
Last-Update: 2019-01-10

--- kbuild-0.1.9998svn3293+dfsg.orig/kBuild/env.sh
+++ kbuild-0.1.9998svn3293+dfsg/kBuild/env.sh
@@ -362,7 +362,7 @@ if test -z "$KBUILD_HOST_ARCH"; then
         alpha)
             KBUILD_HOST_ARCH='alpha'
             ;;
-        aarch32|arm|arm1|arm2|arm3|arm6|armv1|armv2|armv3*|armv4*|armv5*|armv6*|armv7*)
+        aarch32|arm|arm1|arm2|arm3|arm6|armv1|armv2|armv3*|armv4*|armv5*|armv6*|armv7*|armv8*)
             KBUILD_HOST_ARCH='arm32'
             ;;
         aarch64*)
