Description: Docs are non-DFSG
Author: Gianfranco Costamagna <locutusofborg@debian.org>

Last-Update: 2019-01-07

diff --git a/src/kmk/configure.ac b/src/kmk/configure.ac
index 76a38a2..69b09c8 100644
--- a/src/kmk/configure.ac
+++ b/src/kmk/configure.ac
@@ -513,7 +513,7 @@ AS_IF([test "x$make_cv_load" = xno && test "x$user_load" = xyes],
 #AC_CONFIG_FILES([Makefile glob/Makefile po/Makefile.in config/Makefile \
 #                 doc/Makefile w32/Makefile tests/config-flags.pm])
 AC_CONFIG_FILES([Makefile glob/Makefile config/Makefile \
-                 doc/Makefile w32/Makefile tests/config-flags.pm])
+                 w32/Makefile tests/config-flags.pm])
 
 # OK, do it!
 
