Description: Enable support for armv7l to fix FTBFS
Author: Jamie Bennett <jb@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/535093

Index: kbuild-0.1.98svn2318/kBuild/env.sh
===================================================================
--- kbuild-0.1.98svn2318.orig/kBuild/env.sh	2010-03-09 14:36:50.024190343 +0000
+++ kbuild-0.1.98svn2318/kBuild/env.sh	2010-03-09 14:36:50.164189208 +0000
@@ -362,7 +362,7 @@
         hppa64|parisc64)
             KBUILD_HOST_ARCH='hppa64'
             ;;
-        arm|armv4l|armv5tel|armv5tejl)
+        arm|armv4l|armv5tel|armv5tejl|armv7l)
             KBUILD_HOST_ARCH='arm'
             ;;
         alpha)
